execution/evm/vm.go

```diff
@@ -617,8 +618,9 @@ func (vm *VM) execute(callState Interface, eventSink EventSink, caller, callee c
 			if blockNumber > vm.params.BlockHeight {
```

Comment:
The "current" block is not committed so has no hash. Should this be "blockNumber >= vm.params.BlockHeight"

Reply:
This is actually the last block so I think we are good: https://github.com/hyperledger/burrow/blob/develop/execution/contexts/call_context.go#L135

?

Reply:
I think I agree with you remark - we should be passing the height of the block being applied